CadRevealRvmProvider.Tests/Converters/CapVisibilityTests.cs

```diff
@@ -841,6 +839,9 @@ public void CalculateCapVisibility_TestSnoutCapMatch_RotatedCylinders()
 
     [Test]
     [DefaultFloatingPointTolerance(0.01)]
+    [Ignore(
```

Comment:
Maybe create a task for this (if not already done)

Reply:
Good point, I'll create a task and comment in code.